docs/source/daml2ts/index.rst

```diff
@@ -0,0 +1,291 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Typescript!
+###########
+
+daml2ts : Typescript of DAML
+============================
+
+Introduction
+------------
+``daml2ts`` generates `json-api <../json-api/index.html>`_ compatible Typescript code. It accepts compiled DAML modules as input. The Typescript source files it produces contain type, value and function definitions that model the DAML entities it finds.
+
+The code that ``daml2ts`` generates is expressed in terms of the packages `@digital-asset/daml-types <https://github.com/digital-asset/daml/tree/master/language-support/ts/daml-types>`_ and `@mojotech/json-type-validation <https://github.com/mojotech/json-type-validation>`_ packages. Accordingly, every Typescript file generated begins with the following snippet.
```

Comment:
```suggestion
The code that ``daml2ts`` generates is expressed in terms of the packages `@daml/types <https://github.com/digital-asset/daml/tree/master/language-support/ts/daml-types>`_ and `@mojotech/json-type-validation <https://github.com/mojotech/json-type-validation>`_ packages. Accordingly, every Typescript file generated begins with the following snippet.
```
This package has been renamed on master

Reply:
I don't think it's important to mention the `json-type-validation` library. In fact, I consider this an implementation detail and hence mentioning it might be harmful.

Reply:
✔️